src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+import java.util.Random;
+public class Lottery {
+    static ColorSupplier colorSupplier = new ColorSupplier();
+    public static Ball getRandomBall(){
+        Ball randomBall = new Ball();
+        randomBall.setColor(colorSupplier.getRandomColor());
+        int randomIndex = new Random().nextInt(101);
+        randomBall.setNumber(randomIndex);
+        return randomBall;
```

Comment:
Don't create a temporary variable if you can directly return the value. You could return a new `Ball` object inline without assigning it to `randomBall`.